### PR TITLE
apoc.periodic.iterate docs should use parameter

### PR DESCRIPTION
**Pull request by Meatballs1**
Took me a while to get this working until I found:

https://github.com/neo4j-contrib/neo4j-apoc-procedures/issues/205#issuecomment-260084621

### CODE REVIEW

**Comment by jexp** [comment]
Please remind me, did you already sign the Neo4j CLA?
https://neo4j.com/developer/cla/

**Comment by Meatballs1** [comment]
I have done now. Ooh just noticed the PR number  👿

File: docs/periodic.adoc [hunk 1]
```diff
@@ -48,7 +48,7 @@ So if you were to add an `:Actor` label to several million `:Person` nodes, you
 ----
 CALL apoc.periodic.iterate(
 "MATCH (p:Person) WHERE (p)-[:ACTED_IN]->() RETURN p",
-"SET p:Actor", {batchSize:10000, parallel:true})
+"WITH {p} AS p SET p:Actor", {batchSize:10000, parallel:true})
```
Comment (jexp) [inline docs/periodic.adoc]: you don't need that, it's handled automatically.

**Comment by jexp** [comment]
yep cool PR number :)

File: docs/periodic.adoc (hunk 1 above)
Comment (Meatballs1) [reply docs/periodic.adoc]: Is there any differences between 3.0.x and later versions? My query didn't work until I did this and would say that `Variable ``p`` is not defined` etc

Comment (Meatballs1) [reply docs/periodic.adoc]: If so I should maybe PR this against the documentation for the older versions but not against later then

Comment (Meatballs1) [reply docs/periodic.adoc]: Looks like you automatically do this in https://github.com/neo4j-contrib/neo4j-apoc-procedures/commit/35d83dfa50e547c5b40c64877172a7bd361d7e53#diff-f240d1755aedbb2670af0b9ed9776fc9 (3.2?) and remove the documentation. But this information has been removed from the 3.0.x docs:  https://neo4j-contrib.github.io/neo4j-apoc-procedures/index30.html#_apoc_periodic_iterate

**Comment by Meatballs1** [comment]
Looking at 3.0.x https://github.com/neo4j-contrib/neo4j-apoc-procedures/commit/f3899051732d2738455edd53211e7f258f08a1b9 it seems like the changes are there too? Not sure why it didn't work for me without the `with` statement...

**Pull request closed without merging**